integration_test/list_test.go

```diff
@@ -42,3 +47,260 @@ func TestKrewList(t *testing.T) {
 	}
 	// TODO(ahmetb): install multiple plugins and see if the output is sorted
 }
+
+func TestKrewListJSON(t *testing.T) {
+	skipShort(t)
+
+	test, cleanup := NewTest(t)
+	defer cleanup()
+
+	test.Krew("install",
+		"--manifest", filepath.Join("testdata", "foo.yaml"),
+		"--archive", filepath.Join("testdata", "foo.tar.gz")).
+		RunOrFail()
+
+	expected := []byte("[\n    {\n        \"Name\": \"foo\",\n        \"Version\": \"v0.1.0\"\n    }\n]\n")
```

Comment:
why aren't we outputting the index.Plugin object?

this bare `{"Name": ... , "Version": ...}` object is something new that doesn't exist in our ecosystem. let's just give them the index.Plugin as much as we can?

Reply:
>why aren't we outputting the index.Plugin object?

I'm not sure I understand what you mean, index.Plugin is output. Lines 69-77 then take it, confirm its valid JSON and that its values are correct.